website/announcements/api/serializers.py

```diff
@@ -0,0 +1,20 @@
+"""DRF serializers defined by the announcements package."""
+from rest_framework import serializers
+
+from announcements.models import Slide
+
+
+class SlideSerializer(serializers.ModelSerializer):
+    """Slide serializer."""
+
+    class Meta:
+        """Meta class for the serializer."""
+
+        model = Slide
+        fields = (
+            "pk",
+            "title",
+            "content",
```

Comment:
This currently return the full size image, which is probably not what we want because it could be _too_ large.

The slides on the website are resized according to the SLIDE image size setting, the members serializer should have a way to get the right url.